working/augmentation-libraries/feature-specification.md

```diff
@@ -1,6 +1,6 @@
 # Augmentation Libraries
 
-Author: rnystrom@google.com, Version: 1.6 (see [Changelog](#Changelog) at end)
+Author: rnystrom@google.com, Version: 1.7 (see [Changelog](#Changelog) at end)
```

Comment:
You should put your name on here too, unless you don't want any of the credit/blame. :)

Reply:
haha sure